spec/spec_helper.rb

```diff
@@ -55,6 +55,11 @@
   #   EvmSpecHelper.log_ruby_object_usage
   # end
 
+  # everything requires a region
+  config.before(:suite) do
+    MiqRegion.seed
+  end
```

Comment:
IMO, we should leave the DB exactly as it was before starting (i.e. empty), so I'm thinking we also need

```ruby
  # everything requires a region
  config.after(:suite) do
    MiqRegion.delete_all
  end
```

Reply:
I'm pretty sure the database rolls back the changes and even without this it would not be in the database. But, probably doesn't hurt.

Reply:
No I think rollback only happens on each example not at the suite label nor at the all level.